src/test/java/by/onliner/tests/PositiveTests.java

```diff
@@ -0,0 +1,44 @@
+package by.onliner.tests;
+
+import by.onliner.driver.DriverManager;
+import by.onliner.dto.Menu;
+import by.onliner.dto.ShoppingCartModalPopup;
+import by.onliner.page.CatalogPageItem;
+import by.onliner.page.MainPage;
+import by.onliner.page.ShoppingCartPage;
+import by.onliner.util.Assertions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PositiveTests extends AbstractTest{
+
+    @Test(description = "Test for removing item from the shopping cart and checking the cart for emptiness")
+    public void removeItemFromShoppingCartTest() {
+        //given
+        MainPage mainPage = new MainPage();
+        Menu menu = new Menu();
```

Comment:
Isn't the menu should be part of a page?

Reply:
> 

Should, but it should be a part of CatalogPageItem part too... 
And i was thinking about, what i need to do